### DISCUSSION TITLE
Facing problem while uploading deno.

### DISCUSSION THREAD

**Discussion by impixel980** [community/community#143870] [Discussions]
### Select Topic Area

Question

### Body

I'm using to deno to make a discord bot. I cloned a repository . it's working on my local machine. but its not working while i deployed to deno. its showing isolate_environment_failure

**Comment by monwarqx1250** [comment]
its probably because we are missing the environment variable if your project uses the .env. i faced the same issue then I added environment variable then the error is fixed. you can add env variable directly from the deno dashboard or you can specify those by --env name=value while using the deployctl cli

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/143869#discussioncomment-11180557) for reference to why this discussion was closed and locked. Thanks!
